packages/Webkul/Shop/src/Resources/views/customers/forgot-password.blade.php

```diff
@@ -106,13 +106,13 @@ class="primary-button block w-full max-w-[374px] m-0 ml-[0px] mx-auto px-[43px]
         </div>
 
         <p class="mt-[30px] mb-[15px] text-[#6E6E6E] text-xs text-center">
-            @lang('shop::app.customers.forgot-password.footer')
+            {{ trans('shop::app.customers.forgot-password.footer',['currentYear'=> date('Y') ]) }}
```

Comment:
@lang everywhere inside blade files

Reply:
@devansh-webkul Sorry, I can't get it?

Reply:
Can you explain it bit, what you want to say? @devansh-webkul

Reply:
@lang('shop::app.customers.forgot-password.footer', [...])

Reply:
Got it